hack/data-plane.sh

```diff
@@ -198,3 +200,16 @@ function data_plane_setup() {
   data_plane_build_push && k8s
   return $?
 }
+
+function data_plane_dispatcher_setup() {
+  dispatcher_build_push || dispatcher_build_push || fail_test "failed to build dispatcher"
+  header "Creating artifacts"
+
+  echo "Dispatcher image ---> ${KNATIVE_KAFKA_DISPATCHER_IMAGE}"
+
+  ko resolve ${KO_FLAGS} -Rf ${SOURCEV2_DATA_PLANE_CONFIG_DIR} | "${LABEL_YAML_CMD[@]}" >>"${EVENTING_KAFKA_SOURCE_ARTIFACT}"
```

Comment:
Can we include the data plane in the bundle? we cannot release 2 artifacts with the same name.

Reply:
Will do - thanks for catching!